tools/autograd/gen_python_functions.py

```diff
@@ -186,7 +186,7 @@ def should_bind(declaration):
 
     py_torch_functions = group_declarations_by_name(declarations, should_bind)
 
-    env = create_python_bindings(py_torch_functions, has_self=False)
+    env = create_python_bindings(py_torch_functions, has_self=False, is_module=True)
```

Comment:
Why did this change?

Reply:
It removes METH_STATIC on torch_functions.

Without METH_STATIC, those methods can be added to _C via PyModule_AddFunctions.